i18n/about/history.i18n.js

```diff
@@ -125,7 +125,7 @@ export default genI18nMessages({
         pycon2021: {
             title: '2021  是 PyCon Taiwan 十周年',
             content:
-                '面對疫情當前的現在，PyCon Taiwan 可能是今年全球少數能實體舉辦的 PyCon 。今年 PyCon Taiwan 除了傳承十年來的社群交流經驗，我們更做了一些新嘗試，像是 Podcast 及 Young Talent 分享會等。以期 PyCon Taiwan 十周年能促進不同族群間的交流，激盪出不一樣的火花。台灣 Python 年會自許為一個交流平台，我們誠摯邀請每一個台灣 Python 使用者、開發者、推廣者一起多加利用與參與。',
+                '面對疫情當前的現在，PyCon Taiwan 可能是今年全球少數能實體舉辦的 PyCon 。今年 PyCon Taiwan 除了傳承十年來的社群交流經驗，我們更做了一些新嘗試，像是 Podcast 及 Young Inspires 分享會等。以期 PyCon Taiwan 十周年能促進不同族群間的交流，激盪出不一樣的火花。台灣 Python 年會自許為一個交流平台，我們誠摯邀請每一個台灣 Python 使用者、開發者、推廣者一起多加利用與參與。',
```

Comment:
@allen91wu can you help to modify the text that implying we'll have an offline event?